File: MangaCount.Server/Model/EntryModel.cs
using MangaCount.Server.Domain;

namespace MangaCount.Server.Model
{
    public class EntryModel
    {
        public int Id { get; set; }
        public required Manga Manga { get; set; }
        public required int MangaId { get; set; }
        public required int ProfileId { get; set; } // NEW
        public int Quantity { get; set; }
        public string? Pending { get; set; }
        public bool Priority { get; set; }
    }
}

File: mangacount.client/src/app/models/entry.model.ts
import { Manga } from './manga.model';

export interface Entry {
  id: number;
  mangaId: number;
  manga: Manga;
  profileId: number;
  quantity: number;
  pending: string | null;
  priority: boolean;
}

export interface EntryCreateDto {
  id?: number;
  mangaId: number;
  profileId: number;
  quantity: number;
  pending?: string;
  priority: boolean;
}
